safe-global/safe-wallet-web
# fix: Close sidebar drawer when user navigates on mobile

## What it solves

Closes the sidebar on mobile when navigating to a different page

## How to test it

1. Open a Safe on mobile or with a mobile viewport
2. Navigate to a different page
3. Observe the sidebar closes

## Checklist
